### PR TITLE
Add support for OrthoAdam optimizer

### PR DESCRIPTION
**Pull request by klei22**
This optimizer promises with the softmax-1 variation (aka Off-By-One), to highly mitigate magnitude of activation spikes, allowing for straightforward weight and activation PTQ int quantization (and we bet should also make it easier for quantization aware finetuning).

### CODE REVIEW

**Comment by klei22** [comment]
Adding image of the latest comparison, though will have to check that our implementation of OrthoAdam isn't just defaulting to AdamW:

image

**Comment by klei22** [comment]
Added a new ability where pressing qwerty key in the exploration monitor will plot a bar chart based on the number mapping of the (where q = 1, y = 6), this will allow for quick categorical visualization, seen below:

newplot

**Review by gkielian — APPROVED** [review]
Looks good
